### ISSUE TITLE
M2: Make modifications to Activity SLE transitions based on v7

### ISSUE THREAD

**Issue by douglaswyatt** [adityadigitalomnibus/goliath#71]
Not sure how this change got lost - I made the edits on Jan 11th and could swear I emailed or posted it somewhere, but I guess not.  It's a couple of minor changes - a couple of Activity Updates now have required documents, and one activity stage has its name changed.

**Comment by douglaswyatt** [comment]
Not completely correct.  In my testing, I saw two problems.

1) If I add a Final Plan Completed Update while in the Developing Initial Plan stage, it should transition to Executing Plan, but it does not - it stays in Developing Initial Plan.  Note: The Final Plan Completed update does transition to the Executing Plan stage properly if the Activity is in the Developing Final Plan stage.  (See Activity ID J2qeSnx- for an example of how I saw this bug).

2) Initial Idea/Proposal Completed Update does not seem to require an attachment, but it should.



But given how complex the SLE matrix is, I'm curious how these rules are implemented in the code - and wondering if there's a way to look at it to double check it without having to manually test every possible transition myself.
